docs/docs-content/introduction/resource-usage-estimation.md

```diff
@@ -27,16 +27,32 @@ Alternatively, if you continuously deploy a three-node cluster that has 16 CPU p
 
 ## Your Current Usage
 
-Palette and Palette Vertex display your current resource usage on the **Project Overview** page. By default, Palette
-offers 25 kCh per month, but you can always request on-demand compute capacity by clicking **Upgrade now** in the
-bottom-left corner of Palette.
+Palette and Palette VerteX display your current resource usage on the **Project Overview** page. Because we do not
+charge infrastructure costs for Edge and MAAS clusters, the resource usage calculated by Palette and Palette VerteX
+excludes resources used by your Edge or MAAS clusters.
+
```

Comment:
My recommendation is to move the admonition here. That way, we reduce the chances of someone missing it.

Reply:
Hmm, but this one is just a number. The admonition mainly warns people about an unexpected drop in kCh usage, which can only be observed in the graph because that shows month-to-month usage. I can move the admonition to before the screenshot.